src/scenes/index.js

```diff
@@ -10,21 +10,17 @@ import {
 import Home from './Home';
 import TradeHub from './TradeHub';
 
-class Root extends Component {
-  render() {
-    return (
-      <Router>
-        <div>
-          <Route exact path='/' render={(props) => (
-            <Home opts={this.props.opts} />
-          )}/>
-          <Route exact path='/tradehub' render={(props) => (
-            <TradeHub opts={this.props.opts} />
-          )}/>
-        </div>
-      </Router>
-    )
-  }
-}
-
-export default Root;
+export default class Root extends Component {
+  render = () => (
+    <Router>
+      <div>
```

Comment:
Why the `<div>`? Usually you'd put a `<Switch>` here (otherwise the not found component would render at every URL).
Please check [this page](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Switch.md) out for more information.

Reply:
A <Router> may have only one child element. Switch can be used indeed.